### PR TITLE
Fix build on Linux

### PR DESCRIPTION
**Pull request by cesarbs**
Summary of the changes
 - Skip building Microsoft.AspNet.DataProtection.SystemWeb
 - Fix/disable unit tests

#91

### CODE REVIEW

**Comment by cesarbs** [comment]
@muratg @davidfowl @Tratcher @victorhurdugaci

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 DataProtection
```
Comment (Tratcher) [inline README.md]: Revert?

File: makefile.shade [hunk 2]
```diff
@@ -1,45 +1,108 @@
-use assembly='WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```
Comment (Tratcher) [inline makefile.shade]: ??? We should look at your git config autocrlf settings

File: test/Microsoft.AspNet.DataProtection.Extensions.Test/DataProtectionProviderTests.cs [hunk 3]
```diff
@@ -2,8 +2,12 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections;
+using System.Collections.Generic;
 using System.IO;
```
Comment (Tratcher) [inline test/Microsoft.AspNet.DataProtection.Extensions.Test/DataProtectionProviderTests.cs]: You don't need most of these usings do you?

File: makefile.shade [hunk 4]
```diff
@@ -1,45 +1,108 @@
...
+                        @"src\Microsoft.AspNet.DataProtection.SystemWeb\web.config.transform",
+                        @"content\net451\web.config.transform");
+                }
+            }
+        }
+    }
+
+functions
+    @{
+        PackagePart CreatePartFromFile(
```
Comment (davidfowl) [inline makefile.shade]: We should be able to get rid of this and use the new feature that allows packing any file into the nupkg. See https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNet.Server.Kestrel/project.json#L62

File: test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs [hunk 5]
```diff
@@ -77,7 +79,8 @@ public void Encrypt_Decrypt_Tampering_Fails()
             });
         }
 
-        [Fact]
+        [ConditionalFact]
+        [ConditionalRunTestOnlyOnWindows]
```
Comment (Tratcher) [inline test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs]: Comment? What's Windows specific about this one?

File: test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs [hunk 6]
```diff
@@ -94,7 +97,7 @@ public void Encrypt_KnownKey()
             byte[] retVal = encryptor.Encrypt(
                 plaintext: plaintext,
                 additionalAuthenticatedData: aad);
-
+
```
Comment (Tratcher) [inline test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs]: nit: revert

File: test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs (hunk 5 above)
Comment (cesarbs) [inline test/Microsoft.AspNet.DataProtection.Test/Managed/ManagedAuthenticatedEncryptorTests.cs]: To be honest, I'm not sure. I tried getting the same results on Linux but had no success on that. Then I noticed that there are tests with the same name in other classes and those were already marked to run on Windows only.

File: makefile.shade (hunk 4 above)
Comment (cesarbs) [inline makefile.shade]: Fixed.

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**